Visitors rework, anyOf allOf oneOf support (#23)
th2-net/th2-codec-open-api: README.md

### Codec configs:

> * checkUndefinedFields - Enable or Disable warnings for all undefined fields inside object structures, (`true` by default).

Reviewer: ```suggestion
* failOnUndefinedFields - fail on undefined fields inside object structures, (`true` by default).
```
